src/global/Constants.h

```diff
@@ -267,4 +268,6 @@ auto parallel_sort([[maybe_unused]] Args&&... args) {
 using parallel_tag = int;
 }  // namespace ad_utility
 #endif
-static constexpr size_t NUM_SORT_THREADS = 4;
+constexpr size_t NUM_SORT_THREADS = 4;
+constexpr std::string_view EMPH_ON = "\033[1m";
```

Comment:
Maybe a short comment for people who are not super fluent in ANSI escapte sequences.